src/hooks/useExtraSafePaddingBottomStyle.ts

```diff
@@ -0,0 +1,34 @@
+import {useEffect, useMemo, useState} from 'react';
+import {Keyboard} from 'react-native';
+import useStyledSafeAreaInsets from './useStyledSafeAreaInsets';
+import useThemeStyles from './useThemeStyles';
+
+const useExtraSafePaddingBottomStyle = () => {
```

Comment:
Perhaps we can name it something that makes it more clear when it can be used. "extra safe" is a bit ambiguous.